.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: release
+
+# Trigger the workflow when:
+on:
+  # A push occurs to one of the matched tags.
+  push:
+    tags:
+      # Pattern that roughly matches Ledger JS' version tags.
+      # For more details on GitHub Actions' pattern match syntax, see:
+      # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags.
+      - 'v[0-9]+.[0-9]+*'
```

Comment:
Maybe a short note about the release process in the readme would be useful: e.g. "to publish a new version of the package bump the version and push a `v<version>` tag."

Reply:
Added a simple release process doc, we'll improve it once we finish #12.